rosette/src/ForeignFun.cc

```diff
@@ -210,112 +200,112 @@ Ob* ForeignFunction::dispatch(Ctxt* ctxt) {
-             x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27]);
+                (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
+                 x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
+                 x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27]);
             BR;
             CAS(29)
-            (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
-             x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
-             x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28]);
+                (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
+                 x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
+                 x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28]);
             BR;
             CAS(30)
-            (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
-             x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
-             x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28],
-             x[29]);
+                (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
+                 x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
+                 x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28],
+                 x[29]);
             BR;
             CAS(31)
-            (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
-             x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
-             x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28],
-             x[29], x[30]);
+                (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
+                 x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
+                 x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28],
+                 x[29], x[30]);
             BR;
             CAS(32)
-            (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
-             x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
-             x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28],
-             x[29], x[30], x[31]);
-            BR;
-        default: {
-            runtimeError(ctxt, "Exceeded arg limit");
-            ctxt->ret(result);
-            return result;
-        }
+                (x[0], x[1], x[2], x[3], x[4], x[5], x[6], x[7], x[8], x[9], x[10],
+                 x[11], x[12], x[13], x[14], x[15], x[16], x[17], x[18], x[19],
+                 x[20], x[21], x[22], x[23], x[24], x[25], x[26], x[27], x[28],
+                 x[29], x[30], x[31]);
+            BR;
+            default: {
+                         runtimeError(ctxt, "Exceeded arg limit");
+                         ctxt->ret(result);
+                         return result;
+                     }
```

Comment:
These braces aren't mine. In most cases, they were indeed introducing a local variable, but I didn't dig through and figure it out. When I use `clang-format`, it seems to do the best it can with the indent.